src/legilogic_ethereum/ethereum_json_rpc.ml

```diff
@@ -111,18 +121,18 @@ module TransactionParameters = struct
              let yojsoning = {to_yojson;of_yojson}
            end) : (PersistableS with type t := t))
 
-  let of_operation sender operation =
+  let of_operation sender operation value =
```

Comment:
Don't use `some_value_or_null`, pass an option as argument!

Reply:
Yes, this has been done. Not a small work to be done.
But I interpret it as completely implementing the consequence of the correction.